src/driver.rs

```diff
@@ -461,6 +592,11 @@ impl DriverManager {
     }
 }
 
+pub enum DriverProperties {
```

Comment:
Consider the name `DriverType`, as "Properties" usually implies a collection of settings.

Reply:
Thank you, I was initially thinking that properties might be weird. It's renamed now.